m3/customtransports/m3_calc_transport.go

```diff
@@ -20,10 +20,6 @@
 
 package customtransport
 
-import (
-	"sync/atomic"
-)
-
```

Comment:
should probably remove the "It is thread-safe" comment on line 28 below.

Reply:
Good catch, removed.